src/multi_vector_simulator/utils/constants_output.py

```diff
@@ -24,9 +24,10 @@
     ANNUAL_TOTAL_FLOW,
     PEAK_FLOW,
     AVERAGE_FLOW,
+    TOTAL_EMISSIONS,
 )
 
-from multi_vector_simulator.utils.constants_json_strings import LABEL
+from multi_vector_simulator.utils.constants_json_strings import LABEL, EMISSION_FACTOR
```

Comment:
Why do you import the `EMISSION_FACTOR` here?

Reply:
changed